flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/tasks/StreamTaskTest.java

```diff
@@ -315,11 +315,11 @@ public void testStateBackendClosingOnFailure() throws Exception {
 	}
 
 	@Test
-	public void testCancellationNotBlockedOnLock() throws Exception {
```

Comment:
Why was this test removed?

Reply:
Because it was superseded by `SourceStreamTaskTest#testCancellationWithSourceBlockedOnLock()`. The previous test was hacking some fake `StreamTask` implementation that would be behaving exactly as the `SourceStreamTask` (spawning an independent thread, that can block).

Reply:
I see, thanks!